src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/CustomDescription.tsx

```diff
@@ -86,7 +87,9 @@ const MultiSendCustomDataAction = ({ tx, order }: { tx: MultiSendDetails; order:
     >
       <TxDetailsContent>
         <TxInfo>
-          <Bold>Send {humanReadableValue(tx.value)} ETH to:</Bold>
+          <Bold>
+            Send {humanReadableValue(tx.value)} {nativeCoin.name} to:
```

Comment:
Not necessary, but if you can, change this to `fromTokenUnit` and pass the decimals too.